src/components/utils/file.js

```diff
@@ -1,16 +1,23 @@
 import loadImage from 'blueimp-load-image/js/load-image'
 import {canvasToBase64Images, toLossyImageDataUrl} from './canvas.js'
 
-export const fileToBase64 = (file, callback, errorCallback) => {
-  const reader = new FileReader();
-  reader.readAsDataURL(file);
-  reader.onload = function () {
-    callback(reader.result)
-  };
-  reader.onerror = function (error) {
-    console.log('File Reading Error: ', error);
-    errorCallback(error)
-  };
+const decodeBase64 = (image) => {
```

Comment:
it might be worth profiling this for large images. If slow, consider slicing the processing into multiple cycles and turning this function into something async.